### PR TITLE
Merge from stabilization to master

### PR DESCRIPTION
**Pull request by agocke**
/cc @dotnet/roslyn-compiler

### CODE REVIEW

File: build/Targets/VSL.Settings.targets [hunk 1]
```diff
@@ -316,7 +316,7 @@
       $(BuildNumberSuffix.Split('.')[1].PadLeft(2,'0'))
     </BuildNumberPart2>
 
-    <NuGetReleaseVersion>$(RoslynSemanticVersion)</NuGetReleaseVersion>
```
Comment (agocke) [inline build/Targets/VSL.Settings.targets]: Fixing this now -- for master it should stay as `$(RoslynSemanticVersion)`

**Comment by agocke** [comment]
@AlekseyTs Please confirm these are the changes you would expect to flow from stabilization -> master.

**Comment by AlekseyTs** [comment]
@agocke Changes that I expect to flow from stabilization to maser are included into this PR.

**Comment by agocke** [comment]
Note: ETA failure is expected. The original change was cross-cutting.

**Comment by amcasey** [comment]
:+1:

**Pull request merged**